match/lib/match/utils.rb

```diff
@@ -58,6 +58,16 @@ def self.get_cert_info(cer_certificate_path)
       return {}
     end
 
+    def self.check_cert_validity(cer_certificate_path)
+      cert = OpenSSL::X509::Certificate.new(File.binread(cer_certificate_path))
+      now = Time.now.utc
+      if (cert.not_before <=> now) * (now <=> cert.not_after) < 1
+        UI.error("Your certificate '#{File.basename(cer_certificate_path)}' is not valid, please check start and end datetime")
+      else
+        UI.message("Your certificate '#{File.basename(cer_certificate_path)}' is valid")
```

Comment:
```suggestion
        UI.verbose("Your certificate '#{File.basename(cer_certificate_path)}' is valid")
```

I don't think its 100% necessary to put this output - even though it is nice to know 😊 I think it can be implied if it doesn't error that it is valid so I think `verbose` fits best here 👌